packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md

```diff
@@ -160,36 +172,51 @@ This example shows the JSON body for a PUT request to the default Custom Authori
+2. Use the default custom authorization server's authorization endpoint:
 
-    A default Custom Authorization endpoint looks like this where the `${authorizationServerId}` is `default`:
+    > **Note:** See [Authorization servers](/docs/guides/customize-authz-server/overview/) for more information on the types of authorization servers available to you and what you can use them for.
 
-        `https://{yourOktaDomain}/oauth2/default/v1/authorize`
+    A default custom authorization endpoint looks like this where the `${authorizationServerId}` is `default`: `https://{yourOktaDomain}/oauth2/default/v1/authorize`
 
 3. Add the following query parameters to the URL:
 
     * Your OpenID Connect application's `client_id` and `redirect_uri`
-    * The `openid` scope and the scopes that you want to require consent for. In this example, we configured the `phone` scope in the <GuideLink link="../require-consent">previous section</GuideLink>.
+    * The `openid` scope and the scopes that you want to require consent for. In this example, we configured the `phone` scope in the previous section.
     * The response type, which for an ID token is `id_token` and an access token is `token`
 
     > **Note:** The examples in this guide use the [Implicit flow](/docs/concepts/oauth-openid/#implicit-flow), which streamlines authentication by returning tokens without introducing additional steps. This makes it easier to test your configuration. For the [Authorization Code flow](/docs/concepts/oauth-openid/#authorization-code-flow), the response type is `code`. You can then exchange an authorization code for an ID token and/or an access token using the `/token` endpoint.
 
     * Values for `state` and `nonce`, which can be anything
 
-    * Optional. The `prompt` parameter. The standard behavior (if you don't include `prompt` in the request) is to prompt the user for consent if they haven't already given consent for the scope(s). When you include `prompt=consent` in the request, the user is prompted for consent every time, even if they have already given consent. The `consent_method` and the consent for the scope(s) must be set to `REQUIRED`. See the [**Parameter details**](/docs/reference/api/oidc/#parameter-details) section for the `/authorize` endpoint for more information on the supported values for the `prompt` parameter.
+    * Optional. The `prompt` parameter. The standard behavior (if you don't include `prompt` in the request) is to prompt the user for consent if they haven't already given consent for the scope(s). When you include `prompt=consent` in the request, the user is prompted for consent every time, even if they have already given consent. You must set the `consent_method` and the consent for the scope(s) to `REQUIRED`. See the [**Parameter details**](/docs/reference/api/oidc/#parameter-details) section for the `/authorize` endpoint for more information on the supported values for the `prompt` parameter.
```

Comment:
I don't think **Parameter Details** needs to be in bold. (We could also probably drop "...for more information on the supported values for the `prompt` parameter", but I'll leave that up to you.